### PR TITLE
Update boilerplate to work with Rush

### PR DESCRIPTION
**Pull request by craigpalermo**

### CODE REVIEW

File: tools/boilerplate/plop-templates/package/README.md.hbs [hunk 1]
```diff
@@ -1 +1 @@
-# pcln-{{kebabCase componentName}}
+# @priceline/{{kebabCase componentName}}
```
Comment (craigpalermo) [inline tools/boilerplate/plop-templates/package/README.md.hbs:1]: Revert back to `pcln-`

File: tools/boilerplate/plop-templates/package/src/{{componentName}}.stories.js.hbs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import {{componentName}} from '.'
+
+storiesOf('pcln-{{kebabCase componentName}}', module).add('default', () => (
+    <{{componentName}} />
+))
```
Comment (craigpalermo) [inline tools/boilerplate/plop-templates/package/src/{{componentName}}.stories.js.hbs:7]: Update to use CSF format

File: tools/boilerplate/plop-templates/package/test/jest.setup.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { ThemeProvider } from 'styled-components'
+
+import '@testing-library/jest-dom/extend-expect'
+import 'jest-styled-components'
+import renderer from 'react-test-renderer'
+
+import { createTheme } from '../src'
+
+global.rendererCreateWithTheme = (Tree) =>
+  renderer.create(<ThemeProvider theme={createTheme()}>{Tree}</ThemeProvider>)
```
Comment (sdalonzo) [inline tools/boilerplate/plop-templates/package/test/jest.setup.js:11]:
```suggestion
```

File: tools/boilerplate/plop-templates/package/test/jest.setup.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { ThemeProvider } from 'styled-components'
+
+import '@testing-library/jest-dom/extend-expect'
+import 'jest-styled-components'
+import renderer from 'react-test-renderer'
```
Comment (sdalonzo) [inline tools/boilerplate/plop-templates/package/test/jest.setup.js:6]:
```suggestion
```

File: tools/boilerplate/plop-templates/package/src/{{componentName}}.spec.js.hbs [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { fireEvent } from '@testing-library/react'
 
-import {{componentName}} from '../src'
+import {{componentName}} from '.'
```
Comment (sdalonzo) [inline tools/boilerplate/plop-templates/package/src/{{componentName}}.spec.js.hbs:4]: use `{render} from 'testing-library'`

File: tools/boilerplate/plop-templates/package/test/jest.setup.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+import { createTheme } from '../src'
+
+global.rendererCreateWithTheme = (Tree) =>
+  renderer.create(<ThemeProvider theme={createTheme()}>{Tree}</ThemeProvider>)
+
+global.requestAnimationFrame =
+  global.requestAnimationFrame ||
+  function _raf(cb) {
+    return setTimeout(cb, 0)
+  }
```
Comment (sdalonzo) [inline tools/boilerplate/plop-templates/package/test/jest.setup.js:17]: this was a one off for some component, don't need to put it in new components

```suggestion
```

File: tools/boilerplate/plop-templates/package/test/jest.setup.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { ThemeProvider } from 'styled-components'
+
+import '@testing-library/jest-dom/extend-expect'
+import 'jest-styled-components'
+import renderer from 'react-test-renderer'
+
+import { createTheme } from '../src'
```
Comment (sdalonzo) [inline tools/boilerplate/plop-templates/package/test/jest.setup.js:8]: for a new package, this needs to point at `pcln-design-system`

File: tools/boilerplate/plop-templates/package/test/jest.setup.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { ThemeProvider } from 'styled-components'
```
Comment (sdalonzo) [inline tools/boilerplate/plop-templates/package/test/jest.setup.js:2]: just use `ThemeProvider` from `pcln-design-system`, then you don't even need `createTheme()`

File: tools/boilerplate/plop-templates/package/test/testing-library.js [hunk 9]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { render } from '@testing-library/react'
+import { ThemeProvider } from '../src'
```
Comment (sdalonzo) [inline tools/boilerplate/plop-templates/package/test/testing-library.js:4]:
```suggestion
import { ThemeProvider } from 'pcln-design-system'
```

File: tools/boilerplate/plopfile.js [hunk 10]
```diff
@@ -18,7 +20,7 @@ module.exports = function (plop) {
       {
         type: 'addMany',
         templateFiles: 'plop-templates/core/**',
```
Comment (sdalonzo) [inline tools/boilerplate/plopfile.js:22]: no edits to these templates?

**Review by sdalonzo — CHANGES_REQUESTED** [review]
left several comments

File: tools/boilerplate/plop-templates/package/test/jest.setup.js (hunk 8 above)
Comment (craigpalermo) [reply tools/boilerplate/plop-templates/package/test/jest.setup.js]: No longer needed since we removed the global stuff

**Review by sdalonzo — APPROVED** [review]

**Review by gbellini911 — APPROVED** [review]

**Pull request merged**